docs/conceptual/iommu.rst

```diff
@@ -6,93 +6,51 @@
-  want good interrupt distribution across the cores.
-
-* IOMMU enabled in remapping mode. In this mode, the IOMMU is enabled and each device gets it's own IO virtual address space.
-  All devices on Linux register their DMA addressing capabilities (e.g. 32 bit, 40 bit, 44, bit, etc. i.e., how much address
-  space can they natively access). The kernel will then make sure that any address space mapped for DMA is mapped within
-  the DMA addressing limits of the device. Additionally, only address space explicitly mapped by the device is mapped into
-  its virtual address space so an access to an unmapped page will generate an IOMMU page fault. This is generally the
-  default when the IOMMU is enabled.
-
-Advantages and disadvantages
-================================================================
-
-There advantages and disadvantages depending on the devices and platforms you are using.
-
-Advantages
-----------------------------------------------------------------
-
-* Extra security and debuggability. Devices are prevented from access address space they are not allowed to access.
-  This prevents accidental or nefarious device accesses to the platform address space.  Getting an IOMMU page fault on the
-  bad access aids in debugging.
-
-* Device DMA addressing limitations are no longer a problem. The kernel will guarantee that the device will always get
-  DMA virtual addresses within the range of it's DMA addressing limits.  If you have a platform with a large amount of physical
-  memory or if you have a platform where the MMIO aperture is above the DMA addressing limits of the device, some of
-  the system memory may not be accessible or peer to peer DMA is not possible.  This is a common problem in multi-GPU scenarios.
-
-Disadvantages
-----------------------------------------------------------------
-
-* There is added latency for device DMA to the platform address space due to the page table walks required to virtualize the IO address space.
-
-* Peer to Peer DMA requires the kernel config options mentioned above to be enabled in the kernel config.
-  Some older Linux distros do not enable all of those options so peer to peer DMA would not be available in that case.
-  However, it should be noted that not using the IOMMU does not make peer to peer DMA just work.
-  There may be platform limitations as noted above that you could run into unless you know the limitations of your platform and configure it appropriately.
+The input-output memory management unit (IOMMU) provides memory remapping services for IO devices. 
+
+Similar to the memory management unit (MMU) on a CPU, the IOMMU is able to provide a private virtual address space for each device or group of devices connected to a CPU. 
+
+The IOMMU's memory remapping services are used in the following ways:
+
+* To provide private IO space for devices used in a guest virtual machine. 
+* To prevent unauthorized direct memory access (DMA) requests to the memory-managed IO (MMIO).
```

Comment:
MMIO = Memory Mapped IO.  Also, the IOMMU prevents unauthorized DMA access to system memory.  I would say something like:
To prevent unauthorized direct memory access (DMA) requests to system memory and memory mapped IO (MMIO).